festim/exports/derived_quantities/maximum_volume.py

```diff
@@ -4,9 +4,40 @@
 
 
 class MaximumVolume(VolumeQuantity):
+    """
+    Computes the maximum value in a field in a given volume
```

Comment:
Computes the maximum value of a field in a given volume

Reply:
The same for `MinimumVolume`

Reply:
Yeah that is arguably better, I'll update it